qa/suites/basic/health-ok-encrypted.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# DeepSea integration test "suites/basic/health-ok-encrypted.sh"
+#
+# This script runs DeepSea stages 0-3 to deploy a Ceph cluster on all the nodes
+# that have at least one external disk drive. After stage 3 completes, the
+# script checks for HEALTH_OK.
```

Comment:
@jschmid1 Can you update this paragraph to explain in basic terms what the test does?